doc/getting-started.md

```diff
@@ -172,3 +180,43 @@ The default font for material design is [`Roboto`](https://fonts.google.com/spec
 	<MaterialFonts xmlns="using:Uno.Material"
 				   OverrideSource="ms-appx:///Style/Application/MaterialFontsOverride.xaml" />
 	```
+
+## Uno.Cupertino
+Uno Cupertino is an add-on package that lets you apply [Cupertino - Human Interface Guideline styling](https://developer.apple.com/design/human-interface-guidelines) to your application with a few lines of code. 
+
+It features:
+- Color system for both Light and Dark themes
+- Styles for existing WinUI controls like Buttons, TextBox, etc.
+
+### Getting Started
+1. Open an existing Uno project, or create a new Uno project using the `Uno Platform App` template.
+2. In the Solution Explorer panel, right-click on your solution name and select `Manage NuGet Packages for Solution ...`. Choose [`Uno.Cupertino`](https://www.nuget.org/packages/Uno.Cupertino/) package for Uno UWP solution or Choose the [`Uno.Cupertino.WinUI`](https://www.nuget.org/packages/Uno.Cupertino.WinUI) package for Uno WinUI solution. And, select the following projects to include it:
```

Comment:
```suggestion
2. In the Solution Explorer panel, right-click on your solution name and select `Manage NuGet Packages for Solution ...`. Choose [`Uno.Cupertino`](https://www.nuget.org/packages/Uno.Cupertino/) package for Uno UWP solution or choose the [`Uno.Cupertino.WinUI`](https://www.nuget.org/packages/Uno.Cupertino.WinUI) package for Uno WinUI solution. Then, select the following projects to include it:
```

Reply:
Thanks I changed it at another place also in the file for similar issues